kolibri/plugins/user_profile/assets/src/views/ChangeFacility/CreatePassword.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+
+  <div>
+    <h1>{{ $tr('documentTitle') }}</h1>
+    <p>{{ description }}</p>
+    <p>{{ $tr('hint') }}</p>
+    <PasswordTextbox
+      ref="passwordTextbox"
+      data-test="passwordTextbox"
+      :value.sync="formData.password"
+      :isValid.sync="isPasswordValid"
+      :shouldValidate="isFormSubmitted"
+    />
+
+    <p :style=" { color: $themeTokens.annotation }">
+      {{ coreString('rememberThisAccountInformation') }}
+    </p>
+
+    <BottomAppBar>
+      <slot name="buttons">
+        <KButtonGroup>
+          <KButton
+            :primary="false"
+            :text="coreString('backAction')"
+            appearance="flat-button"
+            data-test="backButton"
+            @click="sendBack"
+          />
+          <KButton
+            :primary="true"
+            :text="coreString('continueAction')"
+            data-test="continueButton"
+            :disabled="!isPasswordValid"
+            @click="handleContinue"
+          />
+        </KButtonGroup>
+      </slot>
+    </BottomAppBar>
+  </div>
+
+</template>
+
+
+<script>
+
+  import get from 'lodash/get';
+  import { mapGetters } from 'vuex';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import BottomAppBar from 'kolibri.coreVue.components.BottomAppBar';
+  import PasswordTextbox from 'kolibri.coreVue.components.PasswordTextbox';
+
+  export default {
+    name: 'CreatePassword',
+    metaInfo() {
+      return {
+        title: this.$tr('documentTitle'),
+      };
+    },
+    components: {
+      BottomAppBar,
+      PasswordTextbox,
+    },
+    mixins: [commonCoreStrings],
+    inject: ['changeFacilityService', 'state'],
+    data() {
+      return {
+        formData: {
+          password: '',
+        },
+        isPasswordValid: false,
+        isFormSubmitted: false,
+      };
+    },
+    computed: {
+      ...mapGetters(['session']),
+
+      description() {
+        return this.$tr('description', {
+          username: get(this.session, 'username', ''),
+          targetFacility: get(this.state, 'value.targetFacility.name', ''),
+        });
+      },
+    },
+    methods: {
+      handleContinue() {
+        this.isFormSubmitted = true;
+        if (this.isPasswordValid) {
+          this.sendContinue();
+        } else {
+          this.$refs.passwordTextbox.focus();
+        }
+      },
+      sendContinue() {
+        console.log({
```

Comment:
This doesn't seem like something we want to keep.

Reply:
yep, thanks, that's what review are for. Removed!